v2/internal/project/project.go

```diff
@@ -119,6 +137,73 @@ func (p *Project) Save() error {
 	return os.WriteFile(p.filename, data, 0755)
 }
 
+func (p *Project) setDefaults() {
+
+	if p.Path == "" {
+		p.Path = lo.Must(os.Getwd())
+	}
+	if p.Version == "" {
+		p.Version = "2"
+	}
+	// Create default name if not given
+	if p.Name == "" {
+		p.Name = "wailsapp"
+	}
+	if p.AssetDirectory == "" {
+		p.AssetDirectory = "assets"
```

Comment:
Won't this break all projects that don't use a frontend devserver and rely on inferring of the AssetDirectory? The default until now was empty, which would be to infer it from the assets embed.FS.

Reply:
Just saw this conversation was marked as resolved. I think we should still remove this default value to prevent problems for those users.

Reply:
Not sure if it's just threading issues on GitHub mobile but your reply is to a thread that's unresolved.